README.md

```diff
@@ -1,17 +1,74 @@
 # Rapid Experiments Shared ![CircleCI](https://img.shields.io/circleci/build/github/mozilla/rapid-experiments-shared)
 
-This is a place to define data and schemas used across the rapid experiments
-program.
+This is a place to define data and schemas used across the rapid experiments program.
 
-Any data that moves between systems should have a schema defined here. Schemas
-are defined using Typescript, and automatically converted to JSON Schema. Any
-data that needs to be re-used by multiple systems should be stored here to be
-shared.
+Any data that moves between systems should have a schema defined here. Schemas are defined using
+Typescript, and automatically converted to JSON Schema. Any data that needs to be re-used by
+multiple systems should be stored here to be shared.
+
+## Using the schemas
+
+### Javascript
+
+Install this package
+
+```shell
+$ npm install --save @mozilla/rapid-experiments-shared
+```
+
+Import it.
+
+```js
+import mers from "@mozilla/rapid-experiments-shared";
+```
+
+JSON Schemas can be accessed in `mers.schemas` from JS, or in
+`node_modules/@mozilla/rapid-experiments-shared/schemas` if file system access is more convenient.
+Helper functions to validate objects against those schemas can be found in `mers.typeGuards`.
+
+```js
+import { typeGuards } from "@mozilla/rapid-experiments-shared";
+typeGuards.normandy_checkConsoleLogArguments({});
+/*
```

Comment:
You could link to the validator's documentation if that's useful

Reply:
It's so thoroughly wrapped up right now that it wouldn't really be useful. I plan to add API documentation for everything we export in #14.